core/src/main/scala/org/apache/spark/internal/io/HadoopMapReduceCommitProtocol.scala

```diff
@@ -235,4 +247,20 @@ class HadoopMapReduceCommitProtocol(
       tmp.getFileSystem(taskContext.getConfiguration).delete(tmp, false)
     }
   }
+
+  /**
+   * now just record the file to be delete
+   */
+  override def deleteWithJob(fs: FileSystem,
```

Comment:
nit: code style. see https://github.com/apache/spark/pull/21257/files#diff-d97cfb5711116287a7655f32cd5675cbR132

Reply:
I have changed this code.